packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/testCaseRecorder/takeHarp.yml

```diff
@@ -1,6 +1,6 @@
 languageId: plaintext
 command:
-  version: 4
+  version: 5
```

Comment:
A bit annoying we'll have to bump this one every time we increment command version number, but I think it's not unreasonable: we probably do want to be ensuring that the test case recorder is recording tests using the latest command version

Reply:
Maybe we should have a test to make sure that this test actually contains the latest version?

Reply:
The test case recorder test fails on this one if you don't bump. Are you saying we should add another test?

Reply:
Shouldn't this line always be the last person then?
https://github.com/cursorless-dev/cursorless/blob/b07458bdd94e48dcab97f9b961219f5518fe70eb/packages/cursorless-vscode-e2e/src/suite/testCaseRecorder.test.ts#L76

Reply:
I believe the command that gets captured by recorder is after we do normalisation / upgrade at the start of running the command. So even if we pass 4 there, the command that gets captured has version 5

As to whether that's the right thing to do, I could go either way. I think slightly better to capture normalised version. If we want to be able to record legacy command, we could add a flag to test case recorder to enable that